en/sql/dblink.rst

```diff
@@ -642,23 +642,199 @@ The Query statement below is a Query statement that inquires the remote_t table
+
+.. code-block:: sql
+
+    SELECT ["COLUMN"],["ADD"],["ALTER"] FROM ["TABLE"]@srv1 ;
+    SELECT * FROM dblink(srv1, 'select "COLUMN","ADD","ALTER" from "TABLE" ') AS t(a varchar, b varchar, c varchar );
+
+*   The character used to use reserved words as identifiers in Oracle is the double quotation mark (" ") character.
+
+.. code-block:: sql
+
+    SELECT ["COLUMN"],["ADD"],["ALTER"] FROM ["TABLE"]@srv1 ;
+    SELECT * FROM dblink(srv1, 'select "COLUMN","ADD","ALTER" from "TABLE" ') AS t(a varchar, b varchar, c varchar );
+
+*   The character used to use reserved words as identifiers in MySQL and MariaDB is the backquote (\` \`).
+
+.. code-block:: sql
+
+    SELECT [`COLUMN`],[`ADD`],[`ALTER`] FROM [`TABLE`]@srv1 ;
+    SELECT * FROM dblink(srv1, 'select `COLUMN`,`ADD`,`ALTER` from `TABLE` ') AS t(a varchar, b varchar, c varchar );
+
+
+Constraints
+===========
+
+Common Constraints
+------------------
+
+*   The charset of the remote database used in DBLink supports only utf-8.
+*   Table extension format (object@server)
+        -   Supports only tables, views, and synonyms
+        -   Serial, built-in functions, and stored functions are not supported.
+
+            (Example: sp_func() stored function of remote server (server1) cannot be used in sp_func@server1(arg1, …) format.)
+*   All functions of the SELECT query (built-in functions, stored functions, including SYSDATE), serial-related functions, and system constants all operate locally (if a function or serial is executed in a remote DB, the DBLINK statement must be used)
+    For example, when a select query is made targeting the remote table as below, only the queries in DBLINK() among the rewritten queries are executed in the remote DB.
+
+
+.. code-block:: sql
+
+    SELECT A.*, rownum rn, '' empty, null null_col, SYSDATE
+    FROM t1@srv1 A ;
+    -- rewritten query
+
+    SELECT A.id, A.parentid, A.[text], rownum, '', null, SYS_DATE -- at local
+    FROM ( SELECT [_dbl].id, [_dbl].parentid, [_dbl].[text]
+           FROM DBLINK( srv1 /* '192.168.1.125:33000:remotedb1:dba:*:' */ ,
+                  'SELECT * FROM tree A') AS [_dbl](id integer, parentid integer, [text] varchar(32)) -- at remote
+         ) A (id, parentid, [text])
+
+
+*   All functions of INSERT/UPDATE/DELETE/MERGE queries, serial-related functions, and system constants are executed on the remote server, so be careful when using built-in functions (i.e., CUBRID's built-in functions may not be supported by the remote DBMS or have different usage instructions)
+*   Transaction: Transactions (commit, rollback) between local DB and remote DB are not processed as one transaction. DML (INSERT/UPDATE/DELETE/MERGE syntax) queries in the remote DB are excuted as auto commits separately from transactions in the local DB.
+    As in the example below, when performing a query, data is inserted into the remote DB, but it is rolled back and no data is entered into the local DB.

```

Comment:
```suggestion
    As in the example below, when performing a transaction, data is inserted into the remote DB, but no data is entered into the localDB, because  do rollback.

```